reference/kabusapi/REST_API.md

```
@@ -83,6 +245,30 @@
 |--------|------|---------|
 | `PUT` | `/register` | PUSH 配信銘柄を登録 |
 
+#### PUT `/register`
+
+リクエストボディ (`RequestRegister`)
+
+| Key | Type | Req | Description |
+|-----|------|-----|-------------|
+| Symbols | array<object> |  | 登録する銘柄リスト (最大50) |
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: object

(MD033, no-inline-html)